auth/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export interface Email {
+  value: string;
+  type?: string;
+}
+
+export interface Profile {
```

Comment:
It looks like the basics come from [passport.Profile](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0e9f7eb3a2d623b0ce7d7b95ca8353c8b8e1b784/types/passport/index.d.ts#L138) extended by at least (`nickname`, `user_id`, `picture`) from [auth0](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0e9f7eb3a2d623b0ce7d7b95ca8353c8b8e1b784/types/auth0/index.d.ts#L429) - can we create some union type out of them?

Reply:
IMHO It's easier for users to have a single consistent Profile interface so they don't have to guess which fields are in the profile.
We can extend the basic profile later.